installer/Migration/V5_1_0/lang-string/recruitment.yaml

```diff
@@ -0,0 +1,13 @@
+langStrings:
+  -
+    value: 'View Action History'
+    unitId: view_action_history
+  -
+    value: 'Interviewer'
+    unitId: interviewer
+  -
+    value: 'Interview Title'
+    unitId: interview_title
+  -
+    value: 'Schedule Interview'
+    unitId: schedule_interview
```

Comment:
i only added lang strings.because other PR already have translation files there.so it's easy to add those things after merging previous PR

Reply:
Then keep this PR under draft till that one is merged. and mention which one should be merged before this one for context.